src/views/ReturnDetails.vue

```diff
@@ -35,24 +43,33 @@
                 <p>{{ getProduct(item.productId).productId }}</p>
               </ion-label>
             </ion-item>
-            <ion-item class="product-count">
+            <ion-item :disabled="true" lines="none">
+              <ion-chip outline="true">
+                <ion-icon :icon="locationOutline" />
+                <ion-label>{{ current.locationSeqId }}</ion-label>
+              </ion-chip>
+            </ion-item>
+            <ion-item v-if="isReturnReceivable(current.statusId)" class="product-count">
               <ion-label position="floating">{{ $t("Qty") }}</ion-label>
               <ion-input type="number" min="0" v-model="item.quantityAccepted" />
             </ion-item>
+            <ion-item v-if="!isReturnReceivable(current.statusId)" class="product-count" lines="none">
+              <ion-label>{{ item.quantityAccepted }} {{ $t("received") }}</ion-label>
+            </ion-item>
           </div>
 
           <ion-item class="border-top" v-if="item.quantityOrdered > 0">
-            <ion-button @click="receiveAll(item)" slot="start" fill="outline">
+            <ion-button v-if="isReturnReceivable(current.statusId)" @click="receiveAll(item)" slot="start" fill="outline">
               {{ $t("Receive All") }}
             </ion-button>
             <ion-progress-bar :value="item.quantityAccepted/item.quantityOrdered" />
-            <p slot="end">{{ item.quantityOrdered }}</p>
+            <p slot="end">{{ item.quantityOrdered }} {{ $t("returned") }}</p>
           </ion-item>
         </ion-card>
       </main>
 
       <ion-fab vertical="bottom" horizontal="end" slot="fixed">
-        <ion-fab-button v-if="validStatusChange(current.statusId)" @click="completeShipment">
+        <ion-fab-button v-if="isReturnReceivable(current.statusId)" @click="completeShipment">
```

Comment:
I think we should put this on the parent container. we wouldn't want an empty element floating around the ui

Reply:
Sure sir. Placed it inside a div.

Reply:
@dt2patel please have a look.
PR - https://github.com/hotwax/receiving/pull/138